src/ui/fonts/password_9X9.c

```diff
@@ -0,0 +1,108 @@
+// Converted from password.ttf
```

Comment:
Would be useful to commit this file as well? Eventually those files could be generated at build time from the ttfs.

Reply:
That is a "search and replace" effect. The original file is the one referenced in the normal "a" fonts and that file is lost afaik. This comment should be updated to mention that it is almost a copy of the "font_a" c/h files.

Reply:
Updated the comment.